### ISSUE TITLE
Test status and test output not displayed

### ISSUE THREAD

**Issue by svdHero** [Raathigesh/majestic#207]
## Is this a bug report or a feature request?

Bug Report

## Version Info

- Version of Majestic: v1.7.0
- Version of Jest: v24.9.1
- Version of Node: v14.0.0
- Operating System: Windows 8.1

## Description

I have React app created via `npx create-react-app my-app --template typescript --use-npm` with out-of-the-box Jest functionality. I then added some unit tests, which run fine via `npm test`.

However, when I start a "npx majestic" from my project root directory and click the "Run tests" button the tests run, but I cannot see which tests failed and which succeeded, see screenshot. In fact, there is not any test status assigned to my individual test files.

What's the problem here?

![majestic_bug](https://user-images.githubusercontent.com/24250824/85267863-0f0ef300-b476-11ea-9e31-4ccbb2d01eb8.png)

Also, the directory tree view on the right looks strange. Why does majestic add the whole file path beginning from the file system root `C:\` underneath my project directory "bt_rpe_frontend" from which I started majestic?

**Comment by gregveres** [comment]
Yes I have seen that too. I have found a workaround is usually, if I click on another test and then come back to this test file, the results should show.

**Comment by davidlukerice** [comment]
I'm starting to see this more in my CRA v4 project (which uses jest v26.6.0) after upgrading from CRA v3.
It typically happens when trying to run individual files or tests with `it.only`. Changing to a different file and back doesn't fix it anymore for me, and when I get back to the original file, the top banner is showing the 'test is running' animation.
The console that I'm running majestic from shows the test passed, though.

**Comment by fernand0aguilar** [comment]
I am having the same issue as you guys can see in the screenshot. 
![Screenshot from 2020-12-04 02-20-27](https://user-images.githubusercontent.com/16293264/101109510-5d085500-35d7-11eb-8771-073a50f63d42.png)

Pretty weird. Any solutions?

**Comment by svdHero** [comment]
No, so far not. I have stopped using Majestic because of this. I hope this bug gets fixed soon.

**Comment by robsonsobral** [comment]
I'm under the impression the paths are somehow messed up. On Firefox, I'm getting:

```
Uncaught (in promise) Error: GraphQL error: ENOENT: no such file or directory, open 'C:\Users\sobral\_repositories\foo\E:\_repositories\foo\assets\scripts\forms\toggle-submit-button.unit.test.js'
```

`'C:\Users\sobral\_repositories\foo\'` is a symlink pointing to `'E:\_repositories\foo\'`.

Accessing the folder directly, not by symlink, makes everything work fine.

**Comment by ap1969** [comment]
Same problem: `Uncaught (in promise) Error: GraphQL error: ENOENT: no such file or directory, open 'C:\dev\<package folder>\C:\dev\<other package folder>\src\services\tags\tags.jest.js'`

My set-up involves some scripts in the package folder, and some outside.

No symlinks involved.

Running on windows 10.

In jest.config.js, I have the roots property listing both folders:

```
roots: [
     "<rootDir>",
     pluginFolder
  ],
```

. I suspect this is where the problem lies?

Regards,
Andy

**Comment by mmesarosiuSP** [comment]
@frag997 It's happening for me too and unfortunately I haven't found any solution. In this case the only way to see why a test is failing is by checking the terminal. But the error output in the GUI is what makes Majestic so useful.
